configuration.go

```diff
@@ -50,7 +50,7 @@ var config struct {
 	Broker     broker.Configuration `mapstructure:"broker" toml:"broker"`
 	Server     server.Configuration `mapstructure:"server" toml:"server"`
 	Processing struct {
-		OrgWhiteListFile string `mapstructure:"org_white_list_file" toml:"org_white_list_file"`
+		OrgWhiteListFile string `mapstructure:"org_whitelist" toml:"org_whitelist"`
```

Comment:
maybe it's better to keep it with `_file` since it's a path to the file?

Reply:
+1

Reply:
@Sergey1011010 @JoseLSegura this was actually a bug, the problem with this is #577. It was never loaded from the path in .tomls. 
I'll change the name of the variable in the configs, it makes sense

Reply:
done